src/sage/rings/number_field/number_field.py

```diff
@@ -2639,6 +2639,102 @@ def absolute_field(self, names):
         """
         return NumberField(self.defining_polynomial(), names, check=False, structure=structure.NameChange(self))
 
+    @cached_method
+    def _polred(self, names=None, polredabs=False, isomorphism_map=False):
+        r"""
+        Return ``self`` as an absolute number field defined by a monic polynomial with
+        with reasonably small coefficients by calling PARI ``polredbest`` (or ``polredabs``)
+        and optionally the isomorphism map
+
+        INPUT:
+
+        - ``names`` -- string (default: ``self._names``); name of generator of the absolute field
+        - ``polredabs`` -- boolean (default: ``False``); call ``polredabs`` instead of ``polredbest``
+        - ``isomorphism_map`` -- boolean (default: ``False``); to also return the isomorphism map
+
+        OUTPUT: an absolute number field isomorphic to self defined by a polynomial with
+        smaller discriminant and the isomorphism map
+
+
+        EXAMPLES::
+
+            sage: f = ZZ['x']('x^3 - 12*x^2 + 2*x - 3')
+            sage: NumberField(f, 'a').polred(polredabs=True, isomorphism_map=True)
```

Comment:
Did you add an underscore to the method name?  If so, this test needs to get updated.